elife-00666.xml

```diff
@@ -1797,7 +1797,7 @@ not add all rids to the xref as it does not work. Only add the first one -->
                                     <title>Title of the source code.</title>
                                     <p>Legend of the source code.</p>
                                 </caption>
-                                <media mimetype="application" mime-subtype="xml" xlink:href="elife-00666-fig4-code1.xlsx"/>
+                                <media mimetype="application" mime-subtype="xlsx" xlink:href="elife-00666-fig4-code1.xlsx"/>
```

Comment:
Ah, see what you are saying - but, again, this refers to what we've been doing thus far and keeping it simple. We can review this in a few months, but right now too many other things in the Exeter and eLife camp to change this.